JSDemos/Demos/Map/Markers/AngularJS/index.js

```diff
@@ -30,6 +30,9 @@ DemoApp.controller('DemoController', ($scope) => {
 
   $scope.mapOptions = {
     provider: 'bing',
+    apiKey: {
+      bing: 'Aq3LKP2BOmzWY47TZoT1YdieypN_rB6RY9FqBfx-MDCKjvvWBbT68R51xwbL-AqC',
```

Comment:
Let's add a note that user should not use this key in his app. U can be consulted by a tech writer

Reply:
The customers can't use this api key in his apps because it has domain limitation. The correspond message will appear above the map. I think it's necessary add comment. Also there is nice topic about key specifying for providers in our Map getting started.